backend/backend/settings/base.py

```diff
@@ -86,8 +86,12 @@ def get_required_setting(
 SESSION_EXPIRATION_TIME_IN_SECOND = os.environ.get(
     "SESSION_EXPIRATION_TIME_IN_SECOND", 3600
 )
-SESSION_COOKIE_SECURE = os.environ.get("SESSION_COOKIE_SECURE", True)
-CSRF_COOKIE_SECURE = os.environ.get("CSRF_COOKIE_SECURE", True)
+SESSION_COOKIE_SECURE = CommonUtils.str_to_bool(
+    os.environ.get("SESSION_COOKIE_SECURE", "True")
+)
+CSRF_COOKIE_SECURE = CommonUtils.str_to_bool(
+    os.environ.get("CSRF_COOKIE_SECURE", "True")
+)
```

Comment:
@ritwik-g Might be better to have `False` here as default to cover all use cases.

Will also update the public docs accordingly.

Reply:
cc @athul-rs

Reply:
```suggestion
SESSION_COOKIE_SECURE = CommonUtils.str_to_bool(
    os.environ.get("SESSION_COOKIE_SECURE", "False")
)
CSRF_COOKIE_SECURE = CommonUtils.str_to_bool(
    os.environ.get("CSRF_COOKIE_SECURE", "False")
)
```